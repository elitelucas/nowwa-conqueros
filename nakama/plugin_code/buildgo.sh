docker run --rm -w "/builder" -v "${PWD}:/builder" heroiclabs/nakama-pluginbuilder:3.13.1 build -buildmode=plugin -trimpath -o ./plugin.so
mv ./plugin.so ../plugin.so