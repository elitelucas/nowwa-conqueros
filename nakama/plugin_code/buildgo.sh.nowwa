cd $HOME/nakama/data/plugin_code
docker run --rm -w "/builder" -v "${PWD}:/builder" heroiclabs/nakama-pluginbuilder:2.14.0 build -buildmode=plugin -trimpath -o ./plugin.so
mv plugin.so $HOME/nakama/data/modules/plugin.so
docker-compose restart
