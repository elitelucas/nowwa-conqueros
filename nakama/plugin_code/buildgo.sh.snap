cd $HOME/nakama/data/plugin_code
docker run --rm -w "/builder" -v "${PWD}:/builder" heroiclabs/nakama-pluginbuilder:2.7.0 build -buildmode=plugin -trimpath -o ./plugin_snap.so
